Web/AiiaClient/AiiaHttpClient.cs

```diff
@@ -0,0 +1,95 @@
+﻿#nullable enable
+using System;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Options;
+using Newtonsoft.Json;
+
+namespace Aiia.Sample.Services
+{
+    public class AiiaHttpClient : IAiiaHttpClient
+    {
+        private readonly HttpClient _httpClient;
+
+        public AiiaHttpClient(IOptionsMonitor<SiteOptions> options)
+        {
+            _httpClient = new HttpClient
+            {
+                BaseAddress = new Uri(options.CurrentValue.Aiia.BaseApiUrl)
+            };
+        }
+
+        public async Task<TResponse> CallApi<TRequest, TResponse>(string url,
+            TRequest body,
+            HttpMethod method,
+            string? authScheme,
+            string? authToken)
+        {
+            // prepare request
+            var httpRequestMessage = new HttpRequestMessage(method, url);
+
+            if (body is not null)
```

Comment:
i thought that 'is not null' is only allowed in c# 9.0, but isn't this project still in c#? i mean if it works it works, im just confused a bit

Reply:
I thought it wouldn't work too
I did set the language version to 'latest' and it just compiles and runs 🤷